.github/workflows/ci_test-full.yml

```diff
@@ -62,8 +62,13 @@ jobs:
 
     - name: Install dependencies
       run: |
-        # python -m pip install --upgrade --user pip
-        pip install --requirement requirements/devel.txt --pre --upgrade --quiet --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        python -m pip install --upgrade --user pip
+
+        # Install only mandatory dependencies and check if `import pl_bolts` is successful without any optional packages.
+        pip install --pre --upgrade --quiet .
+        python -c 'import pl_bolts'
```

Comment:
I simply added these lines instead of the complicated tests (I removed them in https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/428/commits/0d9d2d22bdf93f37224fde86597d64901ff9f16f) because `import pl_bolts` imports all the submodules anyway.